pages/common/perl.md

```diff
@@ -21,3 +21,11 @@
 - Run a Perl script in debug mode, using `perldebug`:
 
 `perl -d {{script.pl}}`
+
+- Replace all occurrences of a string in a file with another string, overwriting the file in-place:
+
+`perl -p -i -e 's/{{find}}/{{replace}}/g' {{filename}}`
```

Comment:
in this case, our [e]xpression/program is 's/replacethis/withthat/g'
we are loo[p]ing or iterating through every line of a text file
and making the substitution [i]nplace 

we can't iterate through each line of the file without `-p`